std_msgs/Float64MultiArray[] descriptor